io/net/virt-net/network_virtualization.py

```diff
@@ -148,6 +148,8 @@ def setUp(self):
                'failover' in str(self.name.name):
                 self.cancel("this test is not needed")
         self.local = LocalHost()
+        cmd = "echo 'module ibmvnic +p; func send_subcrq -p' > /sys/kernel/debug/dynamic_debug/control"
+        process.system(cmd, shell=True)
```

Comment:
@smruti77 Can you please use ignore_status and handle the failure case as well ?

Reply:
ok